graphql_compiler/macros/macro_edge/helpers.py

```diff
@@ -140,6 +144,46 @@ def find_target_and_copy_path_to_it(ast):
         return new_ast, target_ast
 
 
+def _get_type_at_target(schema, ast, current_type):
+    """Return type at target or none if there is no target."""
```

Comment:
`none` -> `None` and please clarify what you mean by `target` in the docstring and maybe also in the function name